back/boxtribute_server/graph_ql/definitions/public_api.graphql

```diff
@@ -84,6 +85,35 @@ type TopProductsDimensions {
   category: [DimensionInfo]
 }
 
+type MovedBoxesData implements DataCube {
+  facts: [MovedBoxesResult]
+  dimensions: MovedBoxDataDimensions
+}
+
+"""
+A box can be moved in various ways:
```

Comment:
I analyzed the history table a bit andd what changes actually exist in there that are stock related. Here, the result:
https://docs.google.com/spreadsheets/d/1CLqyYBgCxQ1s5R9h8ySAUPoDqCqXeyoH8dQhzmfVc18/edit?usp=sharing

(Work In Progress)
@pylipp